Description: Change occurrences of /etc/fdprm for /etc/mediaprm
 /etc/mediaprm was formerly named /etc/fdprm. This patch replaces
 occurrences of the old name by the new one, in the file itself and
 in the fdprm manpage.
 .
 It also fixes an indentation typo (this fix comes from ancient times
 and does not seem to be related to any bug).
Author: Matteo Cypriani <mcy@lm7.fr>
Bug-Debian: http://bugs.debian.org/519224
--- a/src/mediaprm
+++ b/src/mediaprm
@@ -1,4 +1,5 @@
-# /etc/fdprm  -  floppy disk parameter table
+# /etc/mediaprm  -  floppy disk parameter table
+# NB: this file was formerly named /etc/fdprm.
 
 ########################################################################
 # Standard linux disk formats. Names are of the form
@@ -774,7 +775,7 @@
 # CMD FD2000 disk drive, a third party floppy disk drive for the
 # Commodore line of computers with double of the capacity:
 "CMDFD2M":
-  DS HD sect=10 cyl=81 ssize=1024 fmt_gap=100 gap=12 swapsides
+ DS HD sect=10 cyl=81 ssize=1024 fmt_gap=100 gap=12 swapsides
 
 # Color computer, sides 1 and 2
 "COCO1":
--- a/doc/setfdprm.1
+++ b/doc/setfdprm.1
@@ -90,7 +90,7 @@
 .PP
 Without any options, \fR\&\f(CWsetfdprm\fR loads the \fIdevice\fR (for example
 \&\fR\&\f(CW\(if/dev/fd0\(is\fR or \fR\&\f(CW\(if/dev/fd1\(is\fR) with a new parameter set with the
-\&\fIname\fR entry found in \fR\&\f(CW\(if/etc/fdprm\(is\fR (usually named 360/360,
+\&\fIname\fR entry found in \fR\&\f(CW\(if/etc/mediaprm\(is\fR (usually named 360/360,
 etc.).  For autodetecting floppy devices, these parameters stay in
 effect until the media is changed. For "fixed parameter" devices, they
 stay in effect until they are changed again.
@@ -104,7 +104,7 @@
 \&\fR\&\f(CW-p\ \fIdevice\ name\fR\&\f(CW\fR\ 
 Permanently loads a new parameter set for the specified auto-configuring
 floppy device for the configuration with \fIname\fR in
-\&\fR\&\f(CW\(if/etc/fdprm\(is\fR. Alternatively, the parameters can be given directly
+\&\fR\&\f(CW\(if/etc/mediaprm\(is\fR. Alternatively, the parameters can be given directly
 from the command line.
 .TP
 \&\fR\&\f(CW-c\ \fIdevice\fR\&\f(CW\fR\ 
--- a/doc/setfdprm.texi
+++ b/doc/setfdprm.texi
@@ -19,7 +19,7 @@
 
 Without any options, @code{setfdprm} loads the @var{device} (for example
 @file{/dev/fd0} or @file{/dev/fd1}) with a new parameter set with the
-@var{name} entry found in @file{/etc/fdprm} (usually named 360/360,
+@var{name} entry found in @file{/etc/mediaprm} (usually named 360/360,
 etc.).  For autodetecting floppy devices, these parameters stay in
 effect until the media is changed. For "fixed parameter" devices, they
 stay in effect until they are changed again.
@@ -34,7 +34,7 @@
 @item -p @var{device name}
 Permanently loads a new parameter set for the specified auto-configuring
 floppy device for the configuration with @var{name} in
-@file{/etc/fdprm}. Alternatively, the parameters can be given directly
+@file{/etc/mediaprm}. Alternatively, the parameters can be given directly
 from the command line.
 
 @item -c @var{device}
